### PR TITLE
Fixing LKL linkification

### PR DESCRIPTION
**Pull request by mspectorgoogle**
LKL likification fixed to correctly add links.

### CODE REVIEW

File: src/appengine/handlers/testcase_detail/show.py [hunk 1]
```diff
@@ -181,7 +182,7 @@ def filter_stacktrace(crash_stacktrace, crash_type, revisions_dict, platform):
     line = html.escape(line, quote=True)
     line = source_mapper.linkify_stack_frame(line, revisions_dict)
 
-    if 'android' in platform:
+    if 'android' in platform or 'lkl' in job_type.lower():
```
Comment (oliverchang) [inline src/appengine/handlers/testcase_detail/show.py:185]: nit: use environment.is_lkl_job(job_type)

File: src/python/platforms/android/kernel_utils.py [hunk 2]
```diff
@@ -46,15 +51,20 @@ def _get_clean_kernel_path(path):
...
     # If this stack frame does not contain a path and line, bail out.
     return stack_frame
 
   path = _get_clean_kernel_path(match.group(1))
   line = match.group(3)
   kernel_prefix = utils.strip_from_left(kernel_prefix, 'kernel/private/')
-  display_path = '/'.join([kernel_prefix, path])
+  display_path = f'{kernel_prefix}/{path}:{line}'
+
+  # If we have a columb number, lets add it to the display path.
```
Comment (oliverchang) [inline src/python/platforms/android/kernel_utils.py:64]: nit: spelling of "column"

**Review by oliverchang — APPROVED** [review]

**Review by inferno-chromium — APPROVED** [review]

**Pull request merged**